vaadin-cookie-consent-flow-demo/src/main/java/com/vaadin/flow/component/cookieconsent/CookieConsentView.java

```diff
@@ -16,6 +17,7 @@
     protected void initView() {
         defaultValues();
         customValues();
+        addCard("Add components to UI", new Label("Add components to UI"));
```

Comment:
Is `<label>` intentional for this? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label

Reply:
`Label` used to be _the_ text component in Vaadin versions prior to V10. In V10+ it's actually a `<label>` element and those aren't typically used on their own, but in association with an `<input>`. Maybe a `Text` or a `Div`/`Span` is preferable nowadays.

Reply:
I am not sure I understand, but you can modify it if you want.

Reply:
This `Label` is used for attaching the useful method to the demo UI. 
same usage can be found [from Grid demo](https://github.com/vaadin/vaadin-grid-flow/blob/master/vaadin-grid-flow-demo/src/main/java/com/vaadin/flow/component/grid/demo/GridDemo.java#L642) 
but, perhaps, @torok-peter , you should add the whole `createCard` method to the block, instead of just one line code.